### DISCUSSION TITLE
Completely Redesigned Version of M³ Available For Testing!

### DISCUSSION THREAD

**Discussion by sircharlo** [sircharlo/meeting-media-manager#2503] [General]
We are excited to announce that after well over a year since [our previous announcement](https://github.com/sircharlo/meeting-media-manager/discussions/1757) and quite a bit of development, **a completely redesigned version of Meeting Media Manager (M³) is now available for testing!** 

This new version introduces significant improvements, including greatly simplified media management. We have eliminated the complex processes of the past, which included media prefixes, remembering what options to click and in what order, and so on. All this has been replaced with an intuitive drag-and-drop interface that will help to streamline the tasks related to being audio-video attendant. 

While we plan to eventually release this version here soon, we are currently inviting users who are comfortable with testing to use the following repo link: [New version download](https://github.com/sircharlo/mmm-refactor/releases/latest)

If you have any comments, questions, or concerns about the new version, please use the discussion [here](https://github.com/sircharlo/mmm-refactor/discussions/2) or the [Issues](https://github.com/sircharlo/mmm-refactor/issues) tab in that repo.

Your feedback will be invaluable in ensuring that any issues are ironed out, that missing features are added, and that the redesigned version functions as intended. Once we confirm its stability and performance, the new version will be released here for all users.

Please keep in mind that our goal for this redesign was twofold: 
1) make the app as simple as possible to use, and 
2) include all features needed to play and manage media for meetings. 

For this reason, some features were intentionally left out for now. If, in the future, certain features are requested by enough of you to show that they would indeed be useful and practical for many, then thought will be put into adding them.

Please keep in mind, **this is an early release**. Please don't use this for live meetings unless you know what you're doing and have a backup plan at the ready.

Thank you for your help in testing out this new version!

**Comment by fschein** [comment]
Hi! Is 24.9.10 the final version that is coming this month?

**Reply by sircharlo** [reply]
24.4.5 is the final version to be released before the big new update.

**Reply by mtdvlpr** [reply]
It's been a couple more due to some last maintenance patches, but the final version will remain in the `24.4.x` range. The big new update starts at `24.6.0` and up.
